cli/diag.c

```diff
@@ -2104,6 +2104,7 @@ static int tlp_inject (int argc, char **argv)
 	}
 	if (num_dwords > 132) {
 		fprintf(stderr, "TLP data cannot exceed 132 dwords \n");
+		free(raw_tlp_dwords);
```

Comment:
This change is not mentioned in the commit message. The free() call looks a bit suspicious but its removal should probably have some comments with it.